cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/helpers/Util.kt

```diff
@@ -383,33 +385,47 @@ object Util {
             }
         }
 
-        // Connect the arguments to parameters
-        val arguments = call.arguments
-        target.parameterEdges.sortWith(Comparator.comparing { it.end.argumentIndex })
+        val callArguments = call.arguments
+        val functionParameters = target.parameters
 
-        var j = 0
-        while (j < arguments.size) {
-            val parameters = target.parameters
-            if (j < parameters.size) {
-                val param = parameters[j]
-                if (param.isVariadic) {
-                    while (j < arguments.size) {
-                        // map all the following arguments to this variadic param
-                        param.prevDFGEdges.addContextSensitive(
-                            arguments[j],
-                            callingContext = CallingContextIn(call),
-                        )
-                        j++
+        var parameterIndex = 0
+
+        for (param in functionParameters) {
```

Comment:
I'm wondering which information are set in the argument `Edge`s on which we can rely here. The names and the index for sure but is there more?